data-products/example_prison_data_product/application/transform.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
```

Comment:
please add unit tests for the new/adjusted methods introduced.

Reply:
yes pls @LavMatt . Add unit test

Reply:
done